### ISSUE TITLE
[bounty] wizardamigos websites

### ISSUE THREAD

**Issue by serapath** [playproject-io/contracting#8]
# `@todo`
* [x] define gig first draft
  * [x] make application form issue
  * [x] make trial phase roadmap issue
  * [x] make gig roadmaping issue
* [ ] finish project
  * [x] publish gig for candidates to apply
  * [x] #5
  * [x] #6
  * [ ] #7

---
 
# `@info`

## who are we?
<details>
<summary>

**The https://playproject.io team is looking to work with independent web ui//ux partner(s)**
*(click here for more details)*
</summary>

1. Everything we do is 100% open source and can be (re)used by anyone in the future.
2. Everyone in our team is independent, self employed and we connect professionally on social media to our global community
3. We give talks at online and physical conferences and support contractors to present their work.
4. Our current project **datodot** is a bleeding edge blockchain project.
5. When our blockchain launches, there will be tokens for everyone who helped along the way.

</details>

## what we are looking for?
**TL;DR**
<details>
<summary>
We are looking for self employed/freelancing web UI/UX designers who know how to use adobe/figma and/or similar tools, and can write raw CSS to create custom css grid or flexbox layouts and also quickly hand code responsiveness and know how to use css properties to make custom themes and are familiar with the latest CSS features, like "container queries" for web components and more.

**=> click here to expand more details**
</summary>

---

1. expert CSS, for example:
    * animations/transitions/transformations during user interaction
    * responsiveness + feature queries + aspect ratio
    * latest css features (e.g. contain queries, containment, @supports)
    * experience using css properties for deriving colors and other things
    * experience with custom css themes for web components
    * no CSS frameworks
    * custom complex css subgrid/flexbox layouts
    * bonus: knowledge of doing "shadow dom" styling (e.g. slot & :root {} selectors)
    * bonus: refine svg graphics and icons and style them
2. expert Adobe CS/XD, Figma (or similar tools), for example:
    * make and change wireframes UI/UX for complex apps/website and UI components
    * create style and theme for apps and UI components based on mood boards
    * create custom **icon sets** or **vector graphics** and/or logos
    * build style guides to show components with theme in different states
3. experience with UX + product development
    * tweak existing UI/UX and make components and user flows more user friendly and lean 
    * research/invent/design new custom UI components based on requirements
    * help us improve the UX of our websites/apps to make them more user friendly
    * come up with new unique components to communicate and enable new unique features

</details>

| gig                              |  `make wireframes + themes for a website`                            | 
|-----------------------------:|:--------------------------------------------------------------------------------|
| application deadline   |  `friday 2022.02.18-23:00(UTC0)`                                           | 
| profile                         | `freelance ux/ui web design`                                                   | 
| budget                        | `200 USD/week` for full time work (`40h/week at 5USD/h`)   | 
| duration                      | 1. `trial phase: 3 days`   2.   `gig phase: 6-12 weeks`           | 

1. we will not accept any applications after the deadline
2. we might make exceptions if you apply late :smirk:, but it has to be before monday
4. we evaluate applications over the weekend `2022.02.18-2022.02.21`
    * **applications will be evaluated against `what we are looking for` (see above)**
5. we notify chosen candidate(s) on monday `2022.02.21` to start the trial phase
6. we then run a paid trial phase with chosen candidate(s)
7. we evaluate the trial phase results at the end
8. we notify the chosen partner(s) to work on the paid gig

*If you have any questions, please leave a comment*
